server/pcioimport.mjs

```diff
@@ -904,7 +904,10 @@ export default async function convertPCIO(content) {
             if(c.args.quantity.type == 'reference')
               quantity = '${' + c.args.quantity.questionId + '}';
             else
-              quantity = c.args.quantity.value;
+              if(c.args.quantity.value == 'all')
```

Comment:
This works, but I personally find the indentation misleading, so I worry it might confuse others in the future. It seems to imply the structure:

```javascript
            if(c.args.quantity.type == 'reference') {
              quantity = '${' + c.args.quantity.questionId + '}';
            } else {
              if(c.args.quantity.value == 'all') {
                quantity = 0;
              } else {
                quantity = c.args.quantity.value;
              }
            }
```

when really it's doing

```javascript
            if(c.args.quantity.type == 'reference') {
              quantity = '${' + c.args.quantity.questionId + '}';
            } else if(c.args.quantity.value == 'all') {
              quantity = 0;
            } else {
              quantity = c.args.quantity.value;
            }
```

but I think joining the "else" and "if" together on the same line would make that more straight-forward, even without using braces.

Reply:
Yes.  Which is why I approved yours. I don't speak JavaScript.  I just make stuff up as I go.